### ISSUE TITLE
Dynamica TabNavigator screen change

### ISSUE THREAD

**Issue by Chatharaju** [react-navigation/react-navigation#3869]
```
const Stylelist = StackNavigator({
    LoginScreen: {
      screen: LoginScreen,
      navigationOptions: ({ navigation }) => ({
        header: null,
      }),
    },
    NotificationsSettingsScreen: {
      screen: NotificationsSettingsScreen,
      navigationOptions: ({ navigation }) => ({
        header: null,
      }),
    },
    SocialSettingsScreen: {
      screen: SocialSettingsScreen,
      navigationOptions: ({ navigation }) => ({
        header: null,
      }),
    },
    Home: {
      screen: TabNavigator({
        RemoteControlScreen: {
          screen: RemoteControlScreen,
          navigationOptions: ({ navigation }) => ({
            header: null,
            tabBarIcon: ({focused }) => <Image source={Images.iconDrop} style={{tintColor: focused ? Colors.lightGreen : null}}/>
          }),
        },
        GetInspiredScreen: {
          screen: GetInspiredScreen,
          navigationOptions: ({ navigation }) => ({
            header: null,
            tabBarIcon: ({focused }) => <Image source={Images.iconGetInspired} style={{tintColor: focused ? Colors.lightGreen : null}}/>
          }),
        },
       ...AppConfig.isUserRegistered ?
        { UserProfileScreen: {
          screen: UserProfileScreen,
          navigationOptions: ({ navigation }) => ({
            header: null,
            tabBarIcon: ({focused }) => <Image source={Images.iconProfile} style={{tintColor: focused ? Colors.lightGreen : null}}/>
          }),
        }, } : {
           CreateAnAccountScreen: {
            screen: CreateAnAccountScreen,
            navigationOptions: ({ navigation }) => ({
              header: null,
              tabBarIcon: ({focused }) => <Image source={Images.iconProfile} style={{tintColor: focused ? Colors.lightGreen : null}}/>
            }),
          },
        }

      },
       {
         tabBarPosition: 'bottom',
         animationEnabled: false,
         swipeEnabled: false,
         showLabel: false,
         tabBarOptions: {
           activeTintColor: 'yellow',
           inactiveTintColor: 'black',
           showLabel: false,
           showIcon: true,
           style: {
             height: (deviceHeight * 5) / 67,
             justifyContent: 'center',
             backgroundColor: Colors.background,
           },
         }
      }),
      navigationOptions: ({ navigation }) => ({
        // title: 'Home',
      }),
    },
  },
  {
    headerMode: 'none',
    initialRouteName: 'SplashScreen',
    navigationOptions: {
      headerStyle: styles.header
    }
  });

```

I have a this navigation structure, I am using tabNavigator inside StackNavigator.
On my TabNavigator, my third tab is based on user login status, i.e., isUserRegistered, UserProfileScreen or CreateAnAccountScreen. Now, I am unable to dynamically load or replace the tab from CreateAnAccountScreen to UserProfileScreen. This means, while am on third tab, upon clicking login, the third tab has to replace with UserProfileTab. 
Basically, I have to re run the app navigation file so that this time the third tab is loaded as  UserProfileScreen instead as CreateAnAccountScreen.

Can someone help to do this.

**Comment by brentvatne** [comment]
hi there! this is a question which is better suited for stackoverflow or discord. see https://reactnavigation.org/help.html for more information on how to get help.
